src/cli.ts

```
@@ -316,6 +331,45 @@ const getOptions = async (argv: any, version?: string) => {
+    // handle sg filters
+    const sgFilter: SgFilter = {
+        includeOrders: undefined,
+        excludeOrders: undefined,
+        includeOwners: undefined,
+        excludeOwners: undefined,
+        includeOrderbooks: undefined,
+        excludeOrderbooks: undefined,
+    };
+    if (cmdOptions.includeOrders) {
+        sgFilter.includeOrders = cmdOptions.includeOrders.map(validateHash);
+        delete cmdOptions.includeOrders;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)